src/main/java/ch/njol/skript/expressions/ExprDamagedItem.java

```diff
@@ -59,7 +75,8 @@ public Class<? extends ItemType> getReturnType() {
 	
 	@Override
 	public String toString(final @Nullable Event e, boolean debug) {
```

Comment:
Can we remove usage of `final` (and anywhere else), and make `Event e` into `Event event`